src/cli/cli.js

```diff
@@ -0,0 +1,34 @@
+import arg from 'arg';
+import { getHelpMenu } from './help_menu.js';
+
+var pjson = require('../../package.json');
```

Comment:
@Simon-Laux where do we store the version number/retrieve it currently?

Reply:
```
import { VERSION, GIT_REF, BUILD_TIMESTAMP } from '../shared/build-info'
```